images/pulp/Containerfile

```
@@ -0,0 +1,16 @@
+ARG PULPCORE_VERSION=nightly
+
+FROM quay.io/centos/centos:stream9
+
+ARG PULPCORE_VERSION
```

Why is the arg repeated here without a value? Does it then inherit the top value? Is that for mult-stage build support? I'd think this could also be simplified down to:
```suggestion
FROM quay.io/centos/centos:stream9

ARG PULPCORE_VERSION=nightly
```

---

Nuance of how they work, they must be re-declared after the FROM.